setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(name='nfldfs',
+      version='0.0.1',
+      author='Brian Doucet',
+      author_email='doucetba@gmail.com',
+      description='A simple package to scrape NFL daily fantasy data.',
+      long_description=long_description,
+      long_description_content_type="text/markdown",
+      url='https://github.com/briandoucet01/nfldfs',
+      packages=['nfldfs'],
```

Comment:
i think you can have a license line in here as well to include the MIT license

Reply:
added license info